config/opensearch_dashboards.yml

```diff
@@ -285,4 +289,7 @@
 # opensearchDashboards.survey.url: "https://survey.opensearch.org"
 
 # Set the value of this setting to true to enable plugin augmentation on Dashboard
-# vis_augmenter.pluginAugmentationEnabled: true
\ No newline at end of file
+# vis_augmenter.pluginAugmentationEnabled: true
+
+# Set the value of this setting to true to enable plugin application config. By default it is disabled.
+# application_config.enabled: false
```

Comment:
nit: Can we put `opensearchDashboards.configIndex` and `application_config.enabled` together as they are related to same feature?

Reply:
Sure. Moved.